tests/unit/test_models.py

```diff
@@ -161,14 +169,9 @@ async def _aiter_bytes():
 
 @pytest.fixture
 def get_pages():
-    p1 = {'links': {'next': 'blah'},
-          'items': [1, 2]}
-    p2 = {'links': {},
-          'items': [3, 4]}
-    responses = [
-        mock_http_response(json=p1),
-        mock_http_response(json=p2)
-    ]
+    p1 = {'links': {'next': 'blah'}, 'items': [1, 2]}
+    p2 = {'links': {}, 'items': [3, 4]}
```

Comment:
For readability, I like the format before this change. However, I don't like it enough to create a special case by disabling yapf.